recipes/r-wgcna/meta.yaml

```diff
@@ -1,19 +1,20 @@
 {% set posix = 'm2-' if win else '' %}
```

Comment:
Bioconda is not building for windows so could you please remove win related info.